hmy/api_backend.go

```diff
@@ -370,7 +370,8 @@ func (b *APIBackend) GetValidatorInformation(
 	computed := availability.ComputeCurrentSigning(
 		snapshot, wrapper,
 	)
-	computed.BlocksLeftInEpoch = shard.Schedule.BlocksPerEpoch() - computed.ToSign.Uint64()
+	beaconChainBlocks := uint64(b.hmy.BeaconChain().CurrentBlock().Header().Number().Int64()) % shard.Schedule.BlocksPerEpoch()
```

Comment:
I think the overflow specifically happens for the multi-bls case where number of blocks to sign is more than schedule.BlocksPerEpoch(). I am not sure how this will fix the problem.

Reply:
may be multiply `shard.Schedule.BlocksPerEpoch()` with number of bls keys? how about `len(wrapper.Validator.SlotPubKeys)`?